components/financial_assistance/app/models/financial_assistance/application.rb

```diff
@@ -1055,6 +1055,15 @@ def create_verification_documents
           %w[Income MEC].collect do |type|
             VerificationType.new(type_name: type, validation_status: 'pending')
           end
+        family_record = Family.where(id: family_id.to_s).first
+        if FinancialAssistanceRegistry.feature_enabled?(:verification_type_income_verification) &&
```

Comment:
We need to do this inside Enroll so that the FAA engine remains isolated and doesn't have to directly use models in Enroll.

Reply:
Alright I did it